### DISCUSSION TITLE
I may be being dumb... but....

### DISCUSSION THREAD

**Discussion by Gigamick** [spatie/laravel-tags#447] [Q&A]
Say I have a model - Post

I will have many "posts" and I want to tag some "posts" with "tag 1" and I want to tag other posts with "tag 2" and I want to tag some posts with both "tag 1" and "tag 2". Is this possible with Spatie tags?

It seems to me that I can only tag the whole model, which isn't very useful to me. But I am 99% sure I am wrong.

Can anyone clarify?

**Comment by LowellTebrero** [comment]
just try to compare the tag name with the user you want to tag, it works in me. ex: $user->name == $tag->name

**Comment by ajuchacko** [comment]
Hey, @Gigamick  your intuition is correct.

use Spatie\Tags\Tag;

$tag = Tag::findOrCreate('my tag');
$tag = Tag::findOrCreate('another tag');

$first_post = Post::first()->attachTag('my tag');
$last_post = Post::get()->last()->attachTag('another tag');

$random_post = Post::find(23)->attachTags(['my tag', 'another tag']);
